source/Weapon.cpp

```diff
@@ -151,6 +151,25 @@ void Weapon::LoadWeapon(const DataNode &node)
 				turn = value;
 			else if(key == "inaccuracy")
 				inaccuracy = value;
+			else if(key == "inaccuracy mode")
+			{
+				if(value == 0)
+					inaccuracyMode = InaccuracyModes::Triangular;
+				else if(value == 1)
+				{
+					inaccuracyMode = InaccuracyModes::Uniform;
+				}
+				else if(value == 2)
```

Comment:
People prefer words over magic numbers, as words can convey intention and meaning. And if they don't, the words aren't right and need work.

Reply:
I addressed this as a question in a comment after the commit that introduced these options (ec6fc033d3524213bd899f7b41b0eaaee4b58d07).  I'll implement it as I outlined.

Reply:
Fixed in c747bc967bf39217463d1fea01ecb03dd4716a0c.